Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1612,7 +1619,8 @@
     <!-- mijenja u korist češće riječi -->
     <RegEx find="([vV])rača" replaceWith="$1raća" />
     <RegEx find="zleči" replaceWith="zliječi" />
-    <RegEx find="(?&lt;!([Oo])bra)zova([lt])" replaceWith="zira$2" />
+    <RegEx find="([nl])izova([nolt])" replaceWith="$1izira$2" />
+    <RegEx find="(?&lt;![Oo]bra)zova([ltno])" replaceWith="zira$1" />
```

Comment:
Anything matched by `([nl])izova([nolt])` will also be matched by `(?<![Oo]bra)zova([ltno])`, making the first of these two RegEX rules unnecessary.